packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -192,7 +192,8 @@ class TestCommand extends FlutterCommand {
         watcher: watcher,
         enableObservatory: collector != null || startPaused,
         startPaused: startPaused,
-        ipv6: argResults['ipv6']);
+        ipv6: argResults['ipv6'],
+        json: machine);
```

Comment:
Nit: maybe put a trailing comma and the closing parens on a new line so future changes only have to affect one line.